simpleStorage/simpleStorage.d.ts

```diff
@@ -0,0 +1,110 @@
+// Type definitions for simpleStorage v0.1.3
+// Project: https://github.com/andris9/simpleStorage
+// Definitions by: Áxel Costas Pena <https://github.com/axelcostaspena>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+declare module andris9_simpleStorage {
```

Comment:
I couldn't use `simpleStorage` because it collided with the exported global var, but thanks to your suggestion, I now used `simplestoragejs`